src/terminal/display.c

```diff
@@ -1020,7 +1035,11 @@ int guac_terminal_display_set_font(guac_terminal_display* display,
     if (new_width != display->width || new_height != display->height)
         guac_terminal_display_resize(display, new_width, new_height);
 
+
     return 0;
 
 }
 
+int get_margin_by_dpi(int dpi) {
+    return dpi * GUAC_TERMINAL_MARGINS / 25.4;
```

Comment:
Maybe `25.4` should be defined as a constant somewhere, which would also help me understand why the value is chosen :-).